Tests/XCDiffCoreTests/Library/StringExtensionsTests.swift

```diff
@@ -0,0 +1,74 @@
+//
```

Comment:
Maybe we can rename it to `String+ExtensionsTests.swift` to be 1 to 1 with the implementation file?